backend-tests/tests/test_devauth.py

```diff
@@ -722,9 +722,16 @@ def do_test_delete_device_ok(self, devs_authsets, user, tenant_token=""):
         assert r.status_code == 401
 
         # verify the device is gone
-        r = devapim.with_auth(utoken).call(
-            "GET", deviceauth.URL_DEVICE, path_params={"id": dev_acc.id}
-        )
+        count = 5
```

Comment:
nit, feel free to ignore. I would rename to indicate that this is the retry count and means the number of seconds (since we have haardcoded sleep(1).

Reply:
It's a copy-paster from another part of the very same test. Let's keep it consistent.